hbase_storage/src/main/java/com/splicemachine/storage/HNIOFileSystem.java

```diff
@@ -197,6 +194,24 @@ public HFileInfo(FileStatus fileStatus) {
             return res;
         }
 
+        @Override
+        public FileInfo[] listDir()
+        {
+            if( !exists() ) return new FileInfo[] {};
+            if( !isDirectory() ) return new FileInfo[] { this };
+            ArrayList<FileInfo> res = new ArrayList<>();
```

Comment:
This probably doesn't matter since it is a local variable, but I would normally declare a variable using the interface type `java.util.List` because it makes it easier to understand the intended usage of it later on (since you will be using standard `List` methods and not something special to `ArrayList`).

Reply:
done